cfpb/consumerfinance.gov
# Form explainers: flag for migration of LE & CD into Wagtail

We are in the process of migrating the two original hard-coded form explainers (for the Loan Estimate & Closing Disclosure forms) into Wagtail. This PR adds a flag that can be used to switch over from the hardcoded templates to the Wagtail versions once they're ready.


## Additions

- `WAGTAIL_FORM_EXPLAINERS` flag


## Changes

- Updates LE + CD routes to use `flagged_wagtail_template_view`


## How to test this PR

1. Check out branch
2. Pull the latest db dump and publish the draft Wagtail `Closing Disclosure` and `Loan Estimate explainer` form explainer pages locally. 
3. Update the slug on the Loan Estimate explainer page to `loan-estimate`. 
4. Navigate to [the loan estimate](http://localhost:8000/owning-a-home/loan-estimate/) and [closing disclosure](http://localhost:8000/owning-a-home/closing-disclosure/) explainer paths and verify that the hard-coded pages are still being displayed (they have a gray Buying a House banner under the mega menu).
5. Enable the `WAGTAIL_FORM_EXPLAINERS` flag
6.  Navigate to the [loan estimate](http://localhost:8000/owning-a-home/loan-estimate/) and [closing disclosure](http://localhost:8000/owning-a-home/closing-disclosure/) explainer paths and verify that the Wagtail versions (sans banner) are now being shown


## Screenshots

### Original CD explainer
| <img width="900" alt="Screen Shot 2022-09-19 at 9 50 28 AM" src="https://user-images.githubusercontent.com/778171/191071070-554d7c35-f235-486e-b2ac-233ef778d417.png"> |
| -- |

### Wagtail CD explainer
| <img width="900" alt="Screen Shot 2022-09-19 at 9 50 49 AM" src="https://user-images.githubusercontent.com/778171/191071091-f0df2c20-1035-43c8-8955-a1a2712881a5.png"> |
| -- |

## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
